cmake/projects/NASM/hunter.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017, Zhuhao Wang
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+if(CMAKE_HOST_WIN32)
+  # NASM only supports a subset of VS compilers on Windows. However, the only
```

Comment:
Is there a variable to check if the user is using MinGW or not? With polly I think we can use `POLLY_MINGW_CMAKE_` but the user may not be using polly.

Otherwise I think we should just disable the building on MinGW toolchain.